packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1146,14 +1146,28 @@ export default class BackupNg {
         $defer.call(xapi, 'deleteVm', snapshot)
       }
 
+      const compress = getJobCompression(job)
+      const pool = snapshot.$pool
+      if (
+        compress === 'zstd' &&
+        pool.restrictions.restrict_zstd_export !== 'false'
+      ) {
+        logger.warning(
+          `Zstd is not supported on the pool ${pool.name_label}, the VM will be exported without compression`,
+          {
+            event: 'task.warning',
+            taskId,
+          }
+        )
+      }
       let xva: any = await wrapTask(
         {
           logger,
           message: 'start snapshot export',
           parentId: taskId,
         },
         xapi.exportVm($cancelToken, snapshot, {
-          compress: getJobCompression(job),
+          compress,
```

Comment:
Should we pass `zstd` if we detected it as not supported?

Reply:
Good question :+1:  No, it should be `false`.